_pages/how-we-work/tools/digital-signatures.md

```diff
@@ -0,0 +1,17 @@
+---
+title: DocuSign - Electronic Signatures
+tag:
+  - eSign
+  - e-sign
+  - signature
+  - digital signature
+  - esignlive
+  - esign live
+  - document
+---
+
+GSA provides access to [DocuSign](https://secureauth.gsa.gov/SecureAuth295/) for staff to sign and request signatures on documents.
+
+To use DocuSign, you ***must*** be on the GSA Wireless network or access it through [VMware Horizon]({{site.baseurl}}/vmware-horizon) or [AnyConnect]({{site.baseurl}}/anyconnect).
```

Comment:
```suggestion
You can access DocuSign off the GSA network by going to [app.docusign.com](https://app.docusign.com).
```

Reply:
Figured this out the other week. Just submitted the feedback form for the team to address, _hopefully_ without requiring being on the network.

Reply:
@cjhskippy @tbaxter-18f Good to commit the change above?

Reply:
I am, I just wasn't clear if we were waiting on something else (cause of the tag)